zbus/src/message.rs

```diff
@@ -520,6 +520,19 @@ impl fmt::Display for Message {
     }
 }
 
+impl TryFrom<Vec<u8>> for Message {
+    type Error = Error;
+
+    fn try_from(bytes: Vec<u8>) -> Result<Self> {
+        Self::from_raw_parts(
+            bytes,
+            #[cfg(unix)]
+            vec![],
```

Comment:
Not sure about this being **generally** correct. Not all users will be loading old messages from a file, like is the case for you. I didn't think of this before. Now I'm wondering if the best solution would be to just expose `Message::from_raw_parts`. :thinking:

Reply:
> Now I'm wondering if the best solution would be to just expose `Message::from_raw_parts`. thinking

If we do that, we should document the method better with all arguments also documented so users know what to pass.

Reply:
I wonder if it would make sense to also use `MessageSequence` as the type of `recv_seq` as that would be self-descriptive.

Reply:
Ah yes, definitely. I missed that it's use u64 as type.

However, now that I think of it, this is only useful for messages received on a `zbus::Connection`. Perhaps it'd be better to provide another wrapper method that doesn't take this argument, `from_bytes`.

BTW, this method should be marked `unsafe` as we can't be sure of `bytes`.